src/editor/codeeditor.h

```diff
@@ -77,15 +115,24 @@ public slots:
     QTimer *mTimer;
     LexerCPP mLexer;
     AddCommentButton *mAddCommentButton;
+    AddCommentTextEdit *mAddCommentTextEdit;
+    CommentWidget *mCommentWidget;
     QLabel *mCurrentCommentLable;
-    int mLinesCount;
+
+    int mLinesCountPrev;
+    int mLinesCountCurrent;
+
     QString mBeginTextState;
+    QVector<AddCommentButton*> mCommentsVector;
+    QSet<AddCommentButton*> mCommentsSet;
```

Comment:
Choose one comment container

Reply:
Set was in the previous version. Forgot to delete it